lib/manageiq/providers/ovirt/redhat_rhevm_probe.rb

```diff
@@ -0,0 +1,16 @@
+require 'manageiq/network_discovery/port_scanner'
+
+module ManageIQ::Providers::Ovirt
+  class RedHatRhevmProbe
```

Comment:
Do we need the `RedHatRhevm` part?  I think the provider namespace would be enough

Reply:
Well we need a class name, because it's autoloaded in [1]. We could use `RedHatRhevm` unless you have another idea. 


[1] https://github.com/ManageIQ/manageiq-network_discovery/blob/master/lib/manageiq-network_discovery.rb#L6